sourcemash/frontend/templates/feed.tpl

```diff
@@ -1,10 +1,8 @@
 <div class="row valign-wrapper">
-    <h2 class="col s8 valign"><%= model.get('title') %></h2>
+    <h2 class="col s8 m9"><%= model.get('title') %></h2>
 
-    <div class="col s2">
-      <button type="submit" class="btn waves-effect waves-light mark-all-read"><i class="mdi-action-done-all"></i></button>
-    </div>
-    <div class="switch subscribe-switch col s2 valign-wrapper"></div>
+    <span title="Mark as Read" class="mark-all-read col s2 m1"><i class="medium mdi-action-done-all"></i></span>
+    <span class="switch subscribe-switch col s2 valign"></span>
```

Comment:
Should this also have ```m2``` so that it adds up to 12? I imagine that ```s2``` does that by default since you don't have it, but seems weird not to have it when both ```col``` classes above this line have it.